xikolo-common/Data/Helper/CoreDataHelper.swift

```diff
@@ -67,4 +67,26 @@ class CoreDataHelper {
         }
     }
 
+    static func clearCoreDataStorage() {
+        clearCoreDataEntity("Course")
+        clearCoreDataEntity("CourseSection")
+        clearCoreDataEntity("CourseItem")
+        clearCoreDataEntity("Content")
+        clearCoreDataEntity("Video")
+        clearCoreDataEntity("RichText")
+        clearCoreDataEntity("Quiz")
+        clearCoreDataEntity("NewsArticle")
```

Comment:
What about `managedObjectContext.entitiesByName` and iterating the dict's keys? That way we won't be able to forget adding entities here.